modules/backend/widgets/Filter.php

```diff
@@ -365,7 +406,7 @@ protected function filterOptionsBySearch($options, $query)
     {
         $filteredOptions = [];
 
-        $optionMatchesSearch = function ($words, $option) {
+        $optionMatchesSearch = function($words, $option) {
```

Comment:
Avoid making inconsistent style changes for the sake of style changes alone please. Same goes for your earlier changes to whitespace between lines.